EIPS/eip-4844.md

```diff
@@ -0,0 +1,564 @@
+
+### Blob gasprice update rule
+
+The blob gasprice update rule is intended to approximate the formula `blob_gas = 2**(excess_blobs / GASPRICE_UPDATE_FRACTION_PER_BLOB)`,
+where `excess_blobs` is the total "extra" number of blobs that the chain has accepted relative to the "targeted" number (`TARGET_BLOB_TXS_PER_BLOCK` per block).
+Like EIP 1559, it's a self-correcting formula: as the excess goes higher, the `blob_gas` increases exponentially, reducing usage and eventually forcing the excess back down.
+
+The block-by-block behavior is roughly as follows.
+If in block `N`, `blob_gas = G1`, and block `N` has `X` blobs, then in block `N+1`, `excess_blobs` increases by `X - TARGET_BLOB_TXS_PER_BLOCK`,
+and so the `blob_gas` of block `N+1` increases by a factor of `2**((X - TARGET_BLOB_TXS_PER_BLOCK) / GASPRICE_UPDATE_FRACTION_PER_BLOB)`.
+Hence, it has a similar effect to the existing EIP 1559, but is more "stable" in the sense that it responds in the same way to the same total usage regardless of how it's distributed.
+
+## Backwards Compatibility
+
+### Blob non-accessibility
+
+This EIP introduces a transaction type that has a distinct mempool version (`BlobTransactionNetworkWrapper`) and execution-payload version (`SignedBlobTransaction`),
+with only one-way convertibility between the two. The blobs are in the `BlobTransactionNetworkWrapper` and not in the `SignedBlobTransaction`;
+instead, they go into the `BeaconBlockBody`. This means that there is now a part of a transaction that will not be accessible from the web3 API.
+
+### Mempool issues
+
+Blob transactions are unique in that they have a variable intrinsic gas cost. Hence, a transaction that could be included in one block may be invalid for the next.
+To prevent mempool attacks, we recommend a simple technique: only propagate transactions whose `gas` is at least twice the current minimum.
+
+Additionally, blob transactions have a large data size at the mempool layer, which poses a mempool DoS risk,
+though not an unprecedented one as this also applies to transactions with large amounts of calldata.
+The risk is that an attacker makes and publishes a series of large blob transactions with fees `f9 > f8 > ... > f1`,
+where each fee is the 10% minimum increment higher than the previous, and finishes it off with a 21000-gas basic transaction with fee `f10`.
+Hence, an attacker could impose millions of gas worth of load on the network and only pay 21000 gas worth of fees.
+
+We recommend a simple solution: both for blob transactions and for transactions carrying a large amount of calldata,
+increase the minimum increment for mempool replacement from 1.1x to 2x, decreasing the number of resubmissions an attacker can do at any given fee level by ~7x.
+
+## Test Cases
+
+Work in progress. Prototype implementations are actively being tested and used to understand the testing surface in more depth.
+
```

Comment:
```suggestion
## Test Cases

TBD
```
Recommend simplifying this so it stands out more as being incomplete and doesn't get overlooked before moving to last call/final.  You don't have to explain why you don't have tests in a draft, you just have to make sure they exist before last call/review (for Core EIPs).